rmc/utils/constraint.py

```diff
@@ -1310,45 +1318,350 @@ def create_context_with_oe(
+    missing_start_or_stop = missing_ht.count()
+    if missing_start_or_stop == 0:
+        return ht
+
+    logger.info(
+        "%i sites are missing either their stop or start AA!", missing_start_or_stop
+    )
+
+    # Get strand from browser HT
+    # TODO: recreate browser HT to include strand annotation
+    browser_ht = gene_model.ht().select("strand")
+    ht = ht.annotate(strand=browser_ht[ht.transcript].strand)
+
+    # Get CDS start/stops from CDS HT
+    transcript_ht = transcript_cds.ht().key_by()
+    transcript_ht = transcript_ht.annotate(
+        cds_start=transcript_ht.interval.start,
+        cds_stop=transcript_ht.interval.end,
+    )
+    ht = ht.annotate(**transcript_ht[ht.transcript])
+    ht = ht.annotate(
+        transcript_start=ht.start_coordinate == ht.cds_start,
+        transcript_stop=ht.stop_coordinate == ht.cds_stop,
+    )
+    transcript_start_stop_fix_ht = fix_transcript_start_stops(ht, overwrite_temp)
+    ht = join_and_fix_aa(ht, transcript_start_stop_fix_ht)
+    ht = ht.checkpoint(
+        f"{TEMP_PATH_WITH_FAST_DEL}/rmc/RMC_results_transcript_start_stop_aa_fix.ht",
+        _read_if_exists=not overwrite_temp,
+        overwrite=overwrite_temp,
+    )
+    region_fix_ht = fix_region_start_stops(ht, context_ht, overwrite_temp)
+    ht = join_and_fix_aa(ht, region_fix_ht)
+    ht = ht.checkpoint(
+        f"{TEMP_PATH_WITH_FAST_DEL}/rmc/RMC_results_all_aa_fix.ht",
+        _read_if_exists=not overwrite_temp,
+        overwrite=overwrite_temp,
+    )
+    return ht
+
+
+def annotate_transcript_globals(ht: hl.Table) -> hl.Table:
+    """
+    Annotate HT globals with transcript information.
+
+    Function is used when reformatting RMC results for browser release.
+    Annotates:
+        - transcripts not searched for RMC
+        - transcripts without evidence of RMC
+        - outlier transcripts
+    :param HT: Input Table. Should be RMC results HT.
+    :return: RMC results HT with updated globals annotations.
+    """
+    # Get transcripts with evidence of RMC
+    rmc_transcripts = hl.literal(ht.aggregate(hl.agg.collect_as_set(ht.transcript)))
+
+    # Get all QC pass transcripts and outlier transcripts
+    qc_pass_transcripts = get_constraint_transcripts(outlier=False)
+    outlier_transcripts = get_constraint_transcripts(outlier=True)
+
+    # Get all transcripts displayed on browser
+    transcript_ht = gene_model.ht()
+    all_transcripts = transcript_ht.aggregate(
+        hl.agg.collect_as_set(transcript_ht.transcript)
+    )
+    transcripts_no_rmc = qc_pass_transcripts.difference(rmc_transcripts)
+    transcripts_not_searched = all_transcripts.difference(qc_pass_transcripts)
+    ht = ht.select_globals()
+    return ht.annotate_globals(
+        transcripts_not_searched=transcripts_not_searched,
+        transcripts_no_rmc=transcripts_no_rmc,
+        outlier_transcripts=outlier_transcripts,
+    )
+
+
+def reformat_annotations_for_release(freeze: int, overwrite_temp: bool) -> None:
```

Comment:
```suggestion
def format_rmc_browser(freeze: int, overwrite_temp: bool) -> None:
```
Suggestion for function name